src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -0,0 +1,221 @@
+package com.denizenscript.depenizen.bukkit.commands.quests;
+
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+
+import me.blackvein.quests.Quest;
+import me.blackvein.quests.Quester;
+import me.blackvein.quests.Quests;
+import me.blackvein.quests.exceptions.InvalidStageException;
+import me.blackvein.quests.util.Lang;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+
+import org.bukkit.entity.Player;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.depenizen.bukkit.bridges.QuestsBridge;
+
+/**
+ * Command integration for Quests
+ * @author PikaMug
+ * @since 2.0.0-b595
+ */
+public class QuestsCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name Quests
+    // @Syntax quests [add/remove/set] (questId:<quest-id>) (stage-no:<#>) (points:<#>) (state:true/false)
+    // @Group Depenizen
+    // @Plugin Depenizen, Quests
+    // @Required 1

```

Comment:
Does this command work with only one argument? Like `- quests add` would do something? If so, what exactly would happen should be more clearly defined. If not, required count here and in the registration should be increased to whatever the actual minimum argument count is.

Reply:
It shouldn't, no. I used the mcMMO command for reference https://github.com/DenizenScript/Depenizen/blob/master/src/main/java/com/denizenscript/depenizen/bukkit/commands/mcmmo/McMMOCommand.java

which also shouldn't and currently has a syntax of 
```
// @Syntax mcmmo [add/remove/set] [levels/xp/xprate/vampirism/hardcore/leader] (skill:<skill>) (state:{toggle}/true/false) (quantity:<#>) (party:<party>)
...
// @Required 1
```

so I assumed that `Required` was not referring to arguments? Anyway, the shortest command would something like

```
// @Usage
// Use to give player 100 Quest Points.
// - quests add points:100
```

whereas a long command would be

```
// @Usage
// Use to force player into specified stage 2 of MyFirstQuest quest.
// - quests set quest_id:custom3 stage_no:2
```

so if we're not counting `add` as an argument, then I believe `Required` should be "1-2".

Reply:
As I said in my non-inline comment: don't use Depenizen code as a reference, it's not properly quality vetted. Use Denizen code for references.
Required is minimum number of arguments to be at all valid.

Reply:
If only you'd said that before the deed haha.

In that case, "1" is correct.

Reply:
In what situation could only 1 argument be input correctly?

Reply:
Please do not mark resolved a comment that you didn't resolve.

Reply:
Such as in `quests add points:100`. Again assuming that `add` is not considered an argument (otherwise it'd be "2-3").

Did I miss something? Seems resolved to me.

Reply:
`add` is considered an argument in that example.